autoload.php

```diff
@@ -28,3 +28,4 @@
 require_once __DIR__ . '/includes/translation/translation-repository.php';
 require_once __DIR__ . '/includes/event-text-snippet.php';
 require_once __DIR__ . '/includes/routes/attendee/list.php';
+require_once __DIR__ . '/includes/routes/attendee/remove-attendee.php';
```

Comment:
I think the pattern we have been following is to not suffix with the entity name. I think it would make sense to do the same here:

```suggestion
require_once __DIR__ . '/includes/routes/attendee/remove.php';
```